### ISSUE TITLE
Werewolf Agility Course Highlight stick (and maybe npc at the end)

### ISSUE THREAD

**Issue by JarikH** [runelite/runelite#6474]
Honestly i know it might sound dumb, but when i'm not paying the best of attention on the werewolf course, i often forget to pick up the stick, yeah i know it might be dumb but it does happen. Maybe it would be possible to make it similar to marks of grace, where it highlights it on the floor if you haven't picked it up yet and makes the next obstacle red untill you pick it up?

Also this is something i haven't seen before so i'm not sure about the possibilities, but maybe a reminder in some shape of form that you have to turn the stick in? Maybe it's possible to highlight the npc when you have a certain item, i'm not sure about that but I do believe highlighting the stick would basically be exactly the same as the marks of grace, but maybe i'm wrong.

**Comment by KasparRosin** [comment]
If you pickup the stick does it appear in your inventory? And if you hand it in, does he take it from inventory?

**Comment by JarikH** [comment]
@KasparRosin Yes, it appears in your inventory, and when you hand it in it dissapears.
